### DISCUSSION TITLE
Website Element Question / Issue

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2037] [Q&A] [answered]
I have an issue with an element on a specific website. See screenshot below.

I am not sure if this is a filter issue, a uBO technical issue requiring a fix to the code or something else.

On the website `https://www.di.fm/` there is an ad placement leftover box that is not affected by uBO. 

Using ABP with just EasyList enabled, the ad placement leftover box was correctly affected.

Initially, I thought it was an EasyList filter issue (https://github.com/easylist/easylist/issues/11226 & https://github.com/easylist/easylist/issues/11158), but they said it is being blocked correctly in their lists and that the problem is something faulty with uBO.

I tested this on Windows 10 in a new, fresh profile in Firefox 98 with uBO 1.41.9b0 using default settings and lists. All filter lists were force updated.

Is there a uBO technical issue causing it to not correctly parse/utilize the EasyList filter they already have for this?

Please help/advise me.

![image](https://user-images.githubusercontent.com/95879668/156963837-ca14de6d-6688-4950-889c-8c4fb58cb0c0.png)

**Comment by mapx-** [comment, marked as answer]
It's nothing faulty, wrong khrin's answer.

In uBO filters was present a ghide filter disabling a generic filter from EasyList.
Now I removed that obsolete filter, just update your lists
https://github.com/uBlockOrigin/uAssets/commit/610e7e71d104cb07634c3e5390b5b32ed90713da

**Reply by ghost** [reply]
Thank you very much!

Working great now!
